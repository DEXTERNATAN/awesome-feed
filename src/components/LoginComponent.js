import React from 'react';

const LoginComponent = ({ setGithubLogin, handleChange }) => (
  <div className="container">
    <div className="jumbotron">
      <h1>Github username</h1>
      <p>
        <input
          type="email"
          className="form-control"
          placeholder="Username"
          onChange={handleChange}
        />
      </p>
      <p>
        <a
          className="btn btn-primary btn-lg"
          onClick={setGithubLogin}
          role="button"
        >
          Send
        </a>
      </p>
    </div>
  </div>
);

export default LoginComponent;
