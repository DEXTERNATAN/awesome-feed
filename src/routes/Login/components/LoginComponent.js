import React from 'react'

const styles = {
  container: {
    marginTop: '5%'
  }
}

const LoginComponent = () => (
  <div className="container" style={styles.container}>
    <div className="jumbotron" />
  </div>
)

export default LoginComponent
