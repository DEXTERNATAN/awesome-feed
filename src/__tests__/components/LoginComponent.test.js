import React from 'react';
import { shallow } from 'enzyme';
import LoginComponent from '../../components/LoginComponent';

describe(`LoginComponent`, () => {
  test(`Snapshot`, () => {
    const props = { setGithubLogin: jest.fn(), handleChange: jest.fn(), onKeyPress: jest.fn() };
    const wrapper = shallow(<LoginComponent {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
