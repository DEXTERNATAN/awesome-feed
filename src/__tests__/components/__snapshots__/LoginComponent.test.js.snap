// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginComponent Snapshot 1`] = `
<div
  className="container"
>
  <div
    className="jumbotron"
  >
    <h1>
      Github username
    </h1>
    <p>
      <input
        className="form-control"
        onChange={[Function]}
        placeholder="Username"
        type="email"
      />
    </p>
    <p>
      <a
        className="btn btn-primary btn-lg"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        Send
      </a>
    </p>
  </div>
</div>
`;
